### PR TITLE
[RFC] Lookup functions from exceptions

### PR DESCRIPTION
**Pull request by tweekmonster**
Displays the most recent exceptions in the QuickFix window to give you a new and exciting way to experience the eye twitches you get from debugging VimL.

- [ ] Tests

### CODE REVIEW

File: autoload/lookup.vim [hunk 1]
```diff
@@ -42,6 +42,117 @@ function! lookup#pop()
...
+              \   'lnum': str2nr(lnum),
+              \   'type': 'I',
+              \ })
+      endif
+
+      let i += 1
+    endfor
+  endfor
+
+  let s:defaults = [
```
Comment (mhinz) [inline autoload/lookup.vim:136]: What is this for? Doesn't look like it should be in here. :>

File: autoload/lookup.vim [hunk 2]
```diff
@@ -107,3 +218,18 @@ endfunction
 function! s:getcurpos() abort
   return [expand('%:p')] + getcurpos()[1:]
 endfunction
+
+" s:exec_lines() {{{1
```
Comment (mhinz) [inline autoload/lookup.vim:222]: Could be renamed to `s:execute()` to make it even more obvious. Would also fit in with the `s:getcurpos()` above it. :)

File: autoload/lookup.vim [hunk 3]
```diff
@@ -107,3 +218,18 @@ endfunction
...
+  endif
+
+  redir => output
+  execute a:cmd
+  redir END
+  return split(output, "\n")
+endfunction
+
+" }}}
+" vim: set ft=vim fdm=marker ts=2 sw=2 et :
```
Comment (mhinz) [inline autoload/lookup.vim:235]: I think this should be just:


```viml
" vim: et sw=2 sts=2 fdm=marker
```

File: autoload/lookup.vim (hunk 1 above)
Comment (tweekmonster) [reply autoload/lookup.vim:136]: Wtf...I'm pretty sure I did this *after* making this PR.  I guess I do weird things late at night.

**Comment by tweekmonster** [comment]
Moving this to a new plugin.

**Pull request closed without merging**